docs/source/release_notes.rst

```diff
@@ -19,6 +19,7 @@ Development Version
   performance.
 * Add option to ``generate_obs_retasking_only`` to prevent computing observations for
   satellites that are continuing their current action.
+* Improve performance of :class:`~bsk_rl.obs.Eclipse` observations by about 95%.
```

Comment:
Excellent change!

Reply:
Do we need to add the change in the access filters here?

Reply:
Sure, I'll note that it's possible to restrict access filters to only act on a certain type of opportunity. The base filter behavior should be the same.